### ISSUE TITLE
Bad resample_spec results for MIRI LRS fixed-slit nodded exposures

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#8485]
_Issue [JP-3624](https://jira.stsci.edu/browse/JP-3624) was created on JIRA by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse):_

Help Desk ticket INC0200735 ([https://stsci.service-now.com/nav_to.do?uri=incident.do?sys_id=21db911a834e8ad05d629da6feaad3fa](https://stsci.service-now.com/nav_to.do?uri=incident.do?sys_id=21db911a834e8ad05d629da6feaad3fa))) reported an issue with the resample_spec results for obs 5 in program 3786, which is a standard MIRI LRS fixed-slit nodded pair of exposures. Obs 11 in the same program is setup identically and the results for that are fine. The problem appears to be related to the way the output WCS grid is setup for the combined output image from resample_spec, such that the two input (cal) images are being combined on top of one another, without the dither/nod being taken into account. A quick look at the WCS information in the exposures for both obs 5 and 11 shows they are very similar, with all the expected ~1.9 arcsec differences in pointing between the 2 exposures in each observation. So the question is why the 2 exposures from obs 11 are being combined properly, with appropriate shifting/alignment due to the dither offset, while obs 5 is not.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698941#comment-698941):_

I'm attaching a screen grab of the two resampled/combined images created for obs 5 (on the left) and obs 11 (on the right). The combined image for obs 11 shows the combined positive spectral trace in the middle of the combined image, with the 2 negative traces (from background subtraction) flanking on either side.

The combined image for obs 5, on the other hand, shows that the two pairs of positive and negative traces from the input cal images have been aligned right on top of one another, without taking the dither/nod into account. This also results in a much narrower combined image. !Screenshot 2024-05-14 at 10.28.00 AM.png!

**Comment by stscijgbot-jp** [comment]
_Comment by [David Law](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dlaw) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698965#comment-698965):_

[Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) Is this problem just with the latest pipeline version or older versions too?  Wondering whether our recent LRS WCS changes might be at fault, or if this is something different.  Also tagging [Greg Sloan](https://jira.stsci.edu/secure/ViewProfile.jspa?name=gcsloan) to be aware.

**Comment by stscijgbot-jp** [comment]
_Comment by [Mihai Cara](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mcara) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698971#comment-698971):_

I am sorry I may have missed this but what are these recent changes?

**Comment by stscijgbot-jp** [comment]
_Comment by [David Law](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dlaw) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698980#comment-698980):_

Build 11.0 includes a change to how the tilt in the LRS spectral trace is dealt with ([https://github.com/spacetelescope/jwst/pull/8411),] but this wouldn't affect anything processed with released pipeline versions.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698982#comment-698982):_

[David Law](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dlaw) The problem happens with both jwst 1.14.0 (B10.2) and the latest B11 dev version.

**Comment by stscijgbot-jp** [comment]
_Comment by [David Law](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dlaw) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=698984#comment-698984):_

[Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) Thanks- sounds like it isn't the new code then.

**Comment by stscijgbot-jp** [comment]
_Comment by [Mihai Cara](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mcara) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=699148#comment-699148):_

[Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) For obs 11 there are two files missing that are present in the ASN:
```java
{
    "expname": "jw03786011001_02101_00001_mirimage_cal.fits",
    "exptype": "target_acquisition",
},
{
    "expname": "jw03786011001_04101_00001_mirimage_cal.fits",
    "exptype": "target_acquisition",
},```
Could you, please, add those files to the data directory?

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=699149#comment-699149):_

[Mihai Cara](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mcara) You don't need those 2 target acquisition images in order to run the pipeline. The Stage 3 pipeline only tries to load "science" and "background" members, not target_acq.

**Comment by stscijgbot-jp** [comment]
_Comment by [Mihai Cara](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mcara) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=699166#comment-699166):_

Ah, it was ModelContainer that was crashing trying to load all these files. My bad. Sorry.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=701806#comment-701806):_

[Mihai Cara](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mcara) New results look great!

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3624?focusedCommentId=701808#comment-701808):_

Fixed by #8511
